rest_docs/rest-catalog-open-api-v0.1.yaml

```diff
@@ -0,0 +1,761 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+---
+openapi: 3.0.3
+info:
+  title: Apache Iceberg REST Catalog API
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.0.1
+servers:
+  - url: http://127.0.0.1:1080
+    description: URL Used for Mock-Server Unit Tests
+# All routes are currently configured using an Authorization header.
+security:
+- BearerAuth: []
+paths:
+  /v1/config:
```

Comment:
Does this v1 mean the Iceberg v1 spec, or the v1 of this Rest API definition?

Reply:
It refers to v1 of the Rest API definition. I think we'd need to support v2 Iceberg spec (as well as v1 spec).

I'll find a place in the `info` section to make a note of this.

Reply:
I updated the `description` on line 28 in the top level `info` block, and bumped the `version` to 1.0.0 to clarify.

Reply:
Is `/<version>/<api>` URI a standard in OpenAPI to separate different API versions? You will also have minor version updates of the API, it does not seem extensible to me. When I think about loading the catalog through endpoint, the endpoint itself could be `https://my.catalog.com/v1`, and the spec itself should not have such version information

Reply:
I wouldn't mind removing it. I'll hold off until other's chime in, who might have more experience in using OpenAPI to version their API spec.

Reply:
We decided to add the `/v1` back in so that clients and servers know which version of the API they are working on.